code/__byond_version_compat.dm

```diff
@@ -2,11 +2,11 @@
 
 //Update this whenever you need to take advantage of more recent byond features
 #define MIN_COMPILER_VERSION 515
-#define MIN_COMPILER_BUILD 1621
+#define MIN_COMPILER_BUILD 1624
 #if (DM_VERSION < MIN_COMPILER_VERSION || DM_BUILD < MIN_COMPILER_BUILD) && !defined(SPACEMAN_DMM)
 //Don't forget to update this part
 #error Your version of BYOND is too out-of-date to compile this project. Go to https://secure.byond.com/download and update.
-#error You need version 515.1621 or higher
+#error You need version 515.1624 or higher
```

Comment:
```suggestion
#error You need version [MIN_COMPILER_VERSION].[MIN_COMPILER_BUILD] or higher
```

I wonder if this works so we don't have ot keep bumping up this string (no issue if it doesn't)

Reply:
it doesn't or we would already